mlflow/tracking/fluent.py

```diff
@@ -283,8 +287,9 @@ def _get_git_commit(path):
     try:
         from git import Repo, InvalidGitRepositoryError, GitCommandNotFound, NoSuchPathError
     except ImportError as e:
-        eprint("Notice: failed to import Git (the Git executable is probably not on your PATH),"
-               " so Git SHA is not available. Error: %s" % e)
+        _logger.error(
+            "Notice: failed to import Git (the Git executable is probably not on your PATH),"
```

Comment:
I think this more of a warning, or info. Can remove Notice.

Reply:
Done!